docs/content/how-tos/rules/FL0075.md

```diff
@@ -0,0 +1,29 @@
+---
+title: FL0075
+category: how-to
+hide_menu: true
+---
+
+# VariableLengthGreaterThanOne (FL0075)
+
+*Introduced in `0.21.0`*
+
+## Cause
+
+Rule to detect usage of 1 letter as variable names, function names, and parameter names.
+
+## Rationale
+
+To ensure that programmers use longer variable names than 1 letter.
```

Comment:
but why should programmers use longer variable names? you have to explain this here, not what you wrote

Reply:
Updated.